server/pom.xml

```diff
@@ -236,6 +236,12 @@
             <groupId>org.codehaus.janino</groupId>
             <artifactId>janino</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.codehaus.jackson</groupId>
+            <artifactId>jackson-mapper-asl</artifactId>
+            <version>1.9.13</version>
```

Comment:
No version number should be added, all version dependencies should be handled in the dependencies module.